### ISSUE TITLE
I can't translate messages automatically

### ISSUE THREAD

**Issue by ruizlenato** [TelegramMessenger/Telegram-iOS#910]
### Checklist
- [X] I am reporting an issue in existing functionality that does not work as intended
- [X] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
The translation system is not working correctly in Brazilian Portuguese.

### Expected Behavior
The translation appears automatically after I click the "translate" button.

### Actual Behavior
 The translation is only shown when I select a target language.

### Steps to Reproduce
1. Select "Portuguese (Brazil)" as the main language of telegram.
2. Try to translate any messages that are not in Portuguese.
3. The translation will not appear until you click "Change language" and select Portuguese.

### Screenshots and Videos 
https://imgur.com/a/EmLaP6o

### Environment
**Device:** `iPhone 7`

**iOS version**: `15.7.1`

**App version:** `9.1.1`

**Comment by ruizlenato** [comment]
I have a hypothesis of what might be happening, apparently Telegram uses the Google Translator API and in the translation target telegram is putting " pt-br" in the language code but google translator unifies Brazilian Portuguese with European Portuguese so the language code is just "pt".

**Comment by DavidLeeMr** [comment]
Yes, I found the same problem, I also faced the same problem when translating from English to Chinese
